dataGenerator.js

```diff
@@ -1,88 +1,77 @@
 
 function descriptionParagraphGenerator(hipsterLorum) {
-  var lowHipString = hipString.toLowerCase().split(' ');
-  var paragraphSize = Math.floor((Math.random()+1)*5).toString()*1// in sentences;
-
-  var hipParagraph='  ';
-
-  for (var i =0; i<paragraphSize; i++) {
-    var sentence = ''
-    var j = 0;
-
-    var sentenceLength  = Math.floor((Math.random()+5)).toString()*1
-    while (j< sentenceLength) {
-
-    randomInt = Math.floor(Math.random()*lowHipString.length)
-    if (j===0 ) {
-      sentence+=' ' + lowHipString[randomInt][0].toUpperCase() + lowHipString[randomInt].slice(1)+ ' ';
-
-    } else if(j === sentenceLength-1){
-
-      sentence+= lowHipString[randomInt]+ '.'
-    } else {
-      sentence+=lowHipString[randomInt] + ' '
+  const lowHipString = hipsterLorum.toLowerCase().split(' ');
+  const paragraphSize = Math.floor((Math.random() + 1) * 5).toString() * 1;// in sentences;
+
+  let hipParagraph = '  ';
+
+  for (let i = 0; i < paragraphSize; i += 1) {
+    let sentence = '';
+    let j = 0;
+
+    const sentenceLength = Math.floor((Math.random() + 5)).toString() * 1;
+    while (j < sentenceLength) {
+      const randomInt = Math.floor(Math.random() * lowHipString.length);
+      if (j === 0) {
+        sentence += ` ${lowHipString[randomInt][0].toUpperCase()}${lowHipString[randomInt].slice(1)} `;
+      } else if (j === sentenceLength - 1) {
+        sentence += `${lowHipString[randomInt]}.`;
+      } else {
+        sentence += `${lowHipString[randomInt]} `;
+      }
```

Comment:
Wow you generated sentences individually, looks great. But may have been easier to just cut the paragraph string and set first letter to uppercase.